test/mocha/release.js

```diff
@@ -8,7 +8,7 @@ function run(command, args, done) {
         process.stdout.write("\0");
     }, 5 * 60 * 1000);
     spawn(command, args, {
-        stdio: "ignore"
+        stdio: [ "ignore", 1, 2 ]
```

Comment:
If this works out I will also be removing that `process.stdout.write("\0");` at 5min intervals, as that was a workaround for Travis CI (& none of the individual scripts from `test/benchmark.js` or `test/jetstream.js` should take more than 10 minutes to be uglified).